src/nodes/strings.js

```diff
@@ -38,10 +38,10 @@ module.exports = {
     return body.length === 2 ? concat([quote, body.slice(1), quote]) : body;
   },
   heredoc: (path, opts, print) => {
-    const { beginning, ending } = path.getValue();
+    const { beging, ending } = path.getValue();
```

Comment:
Is this intentional? lol

Reply:
Haha yeah it is. It was just really long... and it lined up more nicely.